Source/WebKit/Shared/mac/AuxiliaryProcessMac.mm

```diff
@@ -691,7 +691,11 @@ static void populateSandboxInitializationParameters(SandboxInitializationParamet
     }
     setenv("TMPDIR", temporaryDirectory, 1);
 
-    sandboxParameters.addPathParameter("WEBKIT2_FRAMEWORK_DIR", [[webKit2Bundle() bundlePath] stringByDeletingLastPathComponent]);
+    String bundlePath = webKit2Bundle().bundlePath;
+    if (!bundlePath.startsWith("/System/Library/Frameworks"_s))
```

Comment:
Even though you called it out in the commit message, it took me a while to figure out that this is because `WebCore.framework` is a sibling of `WebKit.framework` in local builds rather than within `WebKit.framework/Frameworks` in system builds.